custom/icds_reports/utils.py

```diff
@@ -465,3 +466,19 @@ def indian_formatted_number(number):
         return "".join(r)
     else:
         return 0
+
+
+@quickcache(['domain', 'location_id', 'show_test'], timeout=5 * 60)
+def get_child_locations(domain, location_id, show_test):
+    if location_id:
+        locations = SQLLocation.objects.get(location_id=location_id).get_children()
```

Comment:
would it make sense to include the domain in the filter filter just to prevent any possible bugs where a domain could access another domain's locations?

Reply:
@czue I will add domain to the query.